internal/cli/atlas/deployments/podman/client.go

```diff
@@ -70,13 +98,45 @@ type Client interface {
 	RemoveVolumes(ctx context.Context, names ...string) ([]byte, error)
 	RemoveNetworks(ctx context.Context, names ...string) ([]byte, error)
 	ListContainers(ctx context.Context, nameFilter string) ([]*Container, error)
+	ListImages(ctx context.Context, nameFilter string) ([]*Image, error)
+	PullImage(ctx context.Context, name string) ([]byte, error)
 }
 
 type client struct {
 	debug     bool
 	outWriter io.Writer
 }
 
+func (o *client) Diagnostics(ctx context.Context) *Diagnostic {
+	d := &Diagnostic{
+		Installed: true,
+	}
+
+	err := Installed()
+	if err != nil {
+		d.Installed = false
+		return d
+	}
+
+	info, err := o.machineInspect(ctx)
+	if err != nil { // machine is already present
+		d.MachineFound = false
```

Comment:
I find this a bit confusing, the comment says machine present but then machinefound is set to false?

Reply:
good catch, the comment comes from copy paste, removing